modules/common/development/debug-tools.nix

```diff
@@ -37,10 +37,16 @@ in
           # Performance testing
           speedtest-cli
           iperf
+
+          # Let's have this fixed version according to kernel.
+          # It would be possible to select also latest producing currently (12-3-2024) perf version 6.6.7
+          # linuxPackages_latest.perf
+          linuxKernel.packages.linux_6_1.perf
         ]
-        ++
-        # LuaJIT (which is sysbench dependency) not available on RISC-V
         # TODO Can this be changed to platformPkgs to filter ?
-        lib.optional (config.nixpkgs.hostPlatform.system != "riscv64-linux") sysbench;
+        # LuaJIT (which is sysbench dependency) not available on RISC-V
+        ++ lib.lists.optionals (config.nixpkgs.hostPlatform.system != "riscv64-linux") [sysbench]
```

Comment:
If you are adding just one package, then please use
`lib.optional (config.nixpkgs.hostPlatform.system != "riscv64-linux") sysbench;`

Same goes to below package.

Reply:
Changed back to lib.optional. I had that lib.lists.optionals because first I was adding two packages with single conditional.